src/models/client.ts

```diff
@@ -11,3 +11,73 @@ export interface GetInvoicesArgs {
   limit?: number;
   offset?: number;
 }
+
+interface CreateInvoiceArgsBase {
+  currency: string;
+  price: string | number;
+  orderId?: string | number;
+  expirationTime?: string;
+  itemDesc?: string;
+  itemCode?: string;
+  posData?: string;
+  status?: string;
+  redirectUrl?: string;
+  transactionSpeed?: string;
```

Comment:
This is correct, but I think you can improve this with typescript smartness.
We currently have low,low-medium,medium,high as acceptable values.

Reply:
TS supports enums if that is what you mean